### ISSUE TITLE
Group titles should be h3 headlines

### ISSUE THREAD

**Issue by dominikhorn93** [it-at-m/digiwf-form-builder#5]
**Is your feature request related to a problem? Please describe.**
Group headings cannot be distinguished from normal texts, because the size does not differ. Also, for accessibility reasons, it is important to use proper html elements for sections.

**Describe the solution you'd like**
Use headlines.

**Comment by dominikhorn93** [comment]
@xdoo maybe we should link this issue under a greater epic that is about enabling accessibility... the library we use does not the best job here... I guess we will have to contribute a bit.
